config/serverless.security.yml

```diff
@@ -92,6 +92,9 @@ xpack.fleet.internal.registry.excludePackages: [
 
     # ML integrations
     'dga',
+
+    # Unsupported in serverless
+    'cloud-defend',
```

Comment:
Is cloud-defend is only relevant to security projects? Is it already not possible to install in observability solution for example?

Reply:
And I assume its not available in Elasticsearch projects either?

Reply:
I don't see any way to add integrations to an Elasticsearch project, so I will approve this PR.